setup.cfg

```diff
@@ -0,0 +1,64 @@
+[metadata]
+name = lsst_versions
+description = Calculate package versions following LSST code conventions.
+author = Rubin Observatory Data Management
+url = https://github.com/lsst/lsst_versions
+classifiers =
+    License :: OSI Approved ::  GNU General Public License v3 or later (GPLv3+)
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.8
+version = attr: lsst_versions.__version__
```

Comment:
It may be that this does not do anything useful and is a result of me trying other things. It's definitely the case that utils built fine without this line. The lsst_versions plug in specifically sets the version number for distutils. Maybe setuptools falls back to this if the plugin does not set a version. I need to investigate.